segmentstore/server/src/main/java/io/pravega/segmentstore/server/attributes/SegmentAttributeBTreeIndex.java

```diff
@@ -160,7 +160,8 @@
                         this.storage.openWrite(attributeSegmentName).thenAccept(this.handle::set),
                         ex -> ex instanceof StreamSegmentNotExistsException,
                         () -> this.storage.create(attributeSegmentName, this.config.getAttributeSegmentRollingPolicy(), timer.getRemaining())
-                                          .thenComposeAsync(si -> this.storage.openWrite(attributeSegmentName).thenAccept(this.handle::set), this.executor))
+                                          .thenComposeAsync(si -> this.storage.openWrite(attributeSegmentName)
```

Comment:
We can get rid of this line. `create` returns a handle and that's all we need.

Reply:
oops. I missed that. will fix.